README.md

```diff
@@ -1 +1,158 @@
+    "github.com/kitex-contrib/obs-opentelemetry/tracing"
+)
+
+
+func main()  {
+    p := provider.NewOpenTelemetryProvider(
+        provider.WithServiceName("echo"),
+        provider.WithExportEndpoint("localhost:4317"),
+    )
+    defer p.Shutdown(context.Background())
+
+    svr := echo.NewServer(
+        new(EchoImpl),
+        server.WithSuite(tracing.NewServerSuite()),
+    )
+    if err := svr.Run(); err != nil {
+        klog.Fatalf("server stopped with error:", err)
+    } 	
+}
+
+```
+
+## Client usage
+```go
+import (
+    ...
+    "github.com/kitex-contrib/obs-opentelemetry/provider"
+    "github.com/kitex-contrib/obs-opentelemetry/tracing"
+)
+
+func main(){
+
+    p := provider.NewOpenTelemetryProvider(
+        provider.WithServiceName("echo-client"),
+        provider.WithExportEndpoint("localhost:4317"),
+    )
+    defer p.Shutdown(context.Background())
+    
+    c, err := echo.NewClient(
+        "echo",
+        client.WithSuite(tracing.NewClientSuite()),
+    )
+    if err != nil {
+        klog.Fatal(err)
+    }
+	
+}
+
+```
+
+
+## Kitex Multi Protocol
+> Kitex supports multiple protocols, we can use different suites to initialize opentelemetry observability suites.
+#### Thrift + TTHeader
+```go
+// server
+server.WithSuite(tracing.NewServerSuite())
+// client
+client.WithSuite(tracing.NewClientSuite())
+```
+
+#### GRPC + HTTP2
+```go
+// server
+server.WithSuite(tracing.NewGrpcServerSuite())
+// client
+client.WithSuite(tracing.NewGrpcClientSuite())
+```
+
+
+
+## Tracing associated Logs
+
+#### set logger impl
+```go
+import (
+    kitexlogrus "github.com/kitex-contrib/obs-opentelemetry/logging/logrus"
+)
+
+func init()  {
+    klog.SetLogger(kitexlogrus.NewLogger())
+    klog.SetLevel(klog.LevelDebug)
+
+}
+```
+
+#### log with context
+
+```go
+// Echo implements the Echo interface.
+func (s *EchoImpl) Echo(ctx context.Context, req *api.Request) (resp *api.Response, err error) {
+	klog.CtxDebugf(ctx, "echo called: %s", req.GetMessage())
+	return &api.Response{Message: req.Message}, nil
+}
+```
+
+#### view log
+
+```log
+{"level":"debug","msg":"echo called: my request","span_id":"056e0cf9a8b2cec3","time":"2022-03-09T02:47:28+08:00","trace_flags":"01","trace_id":"33bdd3c81c9eb6cbc0fbb59c57ce088b"}
+```
+
+
+## Example
+
+[Executable Example](https://github.com/cloudwego/kitex-examples/tree/main/opentelemetry)
+
+## Supported Metrics
+
+### RPC Metrics
+
+### Kitex Server
+
+Below is a table of RPC server metric instruments.
+
+| Name | Instrument | Unit | Unit ([UCUM](README.md#instrument-units)) | Description | Status | Streaming |
+|------|------------|------|-------------------------------------------|-------------|--------|-----------|
+| `rpc.server.duration` | Histogram  | milliseconds | `ms` | measures duration of inbound RPC | Recommended | N/A.  While streaming RPCs may record this metric as start-of-batch to end-of-batch, it's hard to interpret in practice. |
```

Comment:
Why don't have a throughput metric?

Reply:
> Why don't have a throughput metric?

[Several other metrics ](https://opentelemetry.io/docs/reference/specification/metrics/semantic_conventions/rpc/#metric-instruments)have not yet been implemented, but if want to calculate qps or throughput, this rpc.server.duration is actually enough.

Because it is a histgoram metric, it will count the number of requests, based on the number of requests we can calculate qps/throughput

Reply:
I see, could you give the description which how to get QPS

Reply:
> description

Like this, we can directly calculate the rate of the count metric of the histogram to calculate the qps

<img width="987" alt="image" src="https://user-images.githubusercontent.com/19374680/159175276-99a3717c-5ad2-4b58-a8d7-d7c0856ed8cb.png">

Reply:
Thanks, I mean that could you add this description in the readme?

Reply:
> Thanks, I mean that could you add this description in the readme?


I explained it in a separate chapter:

![image](https://user-images.githubusercontent.com/19374680/159518644-7df2c83f-8a03-4cbe-89db-b2a27cca380c.png)